service-app/internal/logger/logger.go

```diff
@@ -14,20 +13,50 @@ type Logger struct {
 	SlogLogger *slog.Logger
 }
 
-func NewLogger() *Logger {
-	slogLogger := telemetry.NewLogger("opg-data-lpa-store/getlist")
+func NewLogger(cfg *config.Config) *Logger {
+	slogLogger := telemetry.NewLogger("opg-data-lpa-store/getlist").With(
```

Comment:
Seems to be a stray `opg-data-lpa-store` reference here as well. This should be the service name ("opg-scanning-service") and scaffolds logging that.